lua/dap.lua

```diff
@@ -678,6 +699,17 @@ function M.down()
   end
 end
 
+function M.goto_(line)
+  if session then
+    local source, col
+    if not line then
```

Comment:
Shouldn't the `source` also be set if the `line` is set?

Reply:
Source is optional, since it almost always given by the current stopped frame.